docs/contributing/03-stories/story-16.md

```diff
@@ -0,0 +1,100 @@
+# User Story 16 - Wing for TypeScript
+
+> **Status**: Draft
+
+Anders loves TypeScript, but he also loves the amazing experience Wing offers for building cloud applications.
+
+It's Andres' lucky day because Wing Cloud had just announced that very soon TypeScript developers are able to start leveraging Wing libraries from the comfort of
```

Comment:
```suggestion
It's Anders' lucky day because Wing Cloud had just announced that very soon TypeScript developers are able to start leveraging Wing libraries from the comfort of
```